### PR TITLE
[+] Added Noctua API to search cosmic objects

### PR DESCRIPTION
**Pull request by 0x0is1**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @0x0is1 ;)

Can you resolve this review, please?

File: README.md [hunk 1]
```diff
@@ -1349,6 +1349,7 @@ API | Description | Auth | HTTPS | CORS |
 | [NASA APOD (official API)](https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY) | Official API for getting APOD (Astronomy Image of the Day) images along with metadata | `apiKey` | Yes | Yes |
 | [NASA APOD (unofficial API)](https://apodapi.herokuapp.com/) | API for getting APOD (Astronomy Image of the Day) images along with metadata | No | Yes | Yes |
 | [Newton](https://newton.vercel.app) | Symbolic and Arithmetic Math Calculator | No | Yes | Unknown |
+| [Noctua](https://api.noctuasky.com/api/v1/skysources/name/titan) | Search details of cosmic objects by name | No | Yes | Unknown |
```
[the remark this answers is not in the document]
Comment (0x0is1) [reply README.md:1352]: LGTM. Even it is much more better than mine version. Thank you.

**Review by matheusfelipeog — APPROVED** [review]
Now it's ok ;)

Thanks for the update @0x0is1

**Comment by 0x0is1** [comment]
Dude i was squashing commits, and before i push you merged it. And i was thinking i messed up something.😂

**Pull request merged**